### PR TITLE
#translation: added new German translations

### PR DESCRIPTION
**Pull request by INOPIAE**
I decided not to translate ActiveWorkbook and ActiveSheet.

### CODE REVIEW

File: RetailCoder.VBE/UI/RubberduckUI.de.resx [hunk 1]
```diff
@@ -1183,4 +1183,19 @@ Warnung: Alle eigenen Einstellungen gehen dabei verloren.</value>
   <data name="DisableThisInspection" xml:space="preserve">
     <value>Blockiere diese Untersuchung</value>
   </data>
+  <data name="ImplicitActiveWorkbookReference_" xml:space="preserve">
+    <value>'{0}' ist eine implizierte Referenz auf das 'ActiveWorkbook'</value>
+  </data>
+  <data name="ImplicitActiveSheetReference_" xml:space="preserve">
+    <value>'{0}' ist eine implizite Referenz auf das aktive Tabellenblatt</value>
```
Comment (Vogel612) [inline RetailCoder.VBE/UI/RubberduckUI.de.resx]: Scheint hier nicht konsistent mit Z. 1199
Vorschlag:

> '{0}' ist eine implizite Referenz auf das 'ActiveSheet'

---

Seems to be inconsistent against line 1199

Suggestion: (see above)

**Comment by retailcoder** [comment]
Huzzah! Not translating `ActiveWorkbook` and `ActiveSheet` is probably the right thing to do, since they're defined in the Excel object model, which I don't believe is localized.

File: RetailCoder.VBE/UI/RubberduckUI.de.resx (hunk 1 above)
Comment (retailcoder) [inline RetailCoder.VBE/UI/RubberduckUI.de.resx]: @Vogel612 Good eye! :eyes:

**Comment by INOPIAE** [comment]
I updated the translation.
@retailcoder: At least In German ActiveWorkbook and ActiveSheet are not translated.within the Excel Object model.

**Comment by Vogel612** [comment]
Thank you for your repeated contributions :smile: 

:+1: with addition wonderful from my side. If @retailcoder or @ckuhn203 want the commits squashed: say something now.

**Comment by ckuhn203** [comment]
Nah. I don't mind having a few commits there.

**Pull request merged**